docs/tutorial/persistence.md

```diff
@@ -0,0 +1,73 @@
+3. advertise that you are a global pinner;
+
+
+#### Start your node in global pinning mode
+Starting a node in global pinning mode is done with our standard configuration options (see [start](/docs/tutorial/start-your-node)). The flag which you pass to the command line is `--global-pinning-enable`.
+
+This mode makes your node to listen to PSS (Postal Service over Swarm) messages, containing requests for repair. If your node receives a request for repair and you store the chunk for which repair is requested, your node will re-upload this chunk to its natural location in the network.
+
+#### Persist the files
+Persisting the files is advertised to be done in the `local pinning` feature. However, this is not obligatory; As long as you make sure that you persist the files and have them available at the moment that the recovery mechanism is set in motion (after receiving a PSS message) all is fine.
+
+#### Advertise as a global pinner
+By only starting your node in global pinning mode, nodes will still not know that they need to reach out to you for repair. You must somehow report that you are globally pinning the files (and this information must be picked up by any person who downloads the content you are globally pinning). 
+
+The Swarm team envisions multiple ways how this is possible: via smart-contracts, via special gateways to the Swarm network, via Manifests, as data in the root chunk, via a feed or many other ways. 
+
+We deliberately didn't force a specific pattern to the user, and we also don't aim to do so in the future. However, when a certain pattern to advertise yourself as a global pinner becomes the de-facto standard, we will report it here.
+
+:::info
+As a global pinner, you don't need to advertise your full overlay address. You need to advertise the first n characters of your overlay address, where n increases if the number of nodes in the network increases.
+:::
+
+TODO: exact value
+
+
+### Request files which are globally pinned
+To make use of the global pinning feature (and request a repair if a chunk not found), you need to pass a `targets` query parameter to your upload request. The value of this parameter is a reference to the address of the global pinner node. You don't need to pass his whole address, just the first n characters is sufficient (where n increases if the number of nodes in the network increases). 
+TODO: exact value of n.
+
+An example of a request to download a file with the targets query parameter passed in:
+
+```sh
+curl http://localhost:8080/files/3b2791985f102fe645d1ebd7f51e522d277098fcd86526674755f762084b94ee?targets=<target comes here>
+```
+
+If your chunk is not found in the network, but you sent a request to the passed-in target for repair then you get a `202` response back, indicating that the request has been accepted for processing.
+
+You should now retry to download the chunk (possibly with another target, in case your first target didn't process the request). 
```

Comment:
After few seconds, You can retry to download the chunk again and this time the network will respond back with the repaired chunk.

If you still don't get the chunk back, try downloading the chunk with another target.

Reply:
Thx, better